source/MaterialXView/Viewer.cpp

```diff
@@ -2467,3 +2467,35 @@ void Viewer::updateAlbedoTable()
     glBindFramebuffer(GL_DRAW_FRAMEBUFFER, 0);
     glDrawBuffer(GL_BACK);
 }
+
+void Viewer::renderScreenSpaceQuad(MaterialPtr material)
+{
+
+    ng::MatrixXf positions(3, 4);
```

Comment:
Since we're aiming to reduce our dependencies on this version of NanoGUI over time (eventually moving to a newer version that doesn't use Eigen), I would recommend constructing these matrices as constant instances of MaterialX classes, and converting them to NanoGUI format only when they're passed to NanoGUI functions.

Reply:
Updated to use mx::Mesh and other classes.